web/templates/pages/list_weblog.php

```diff
@@ -105,10 +79,14 @@
 		</div>
 	</header>
 
-	<button type="button" class="button button-secondary button-circle button-floating button-floating-top js-to-top" title="<?=_('Top');?>">
+	<a
+		href="#top"
```

Comment:
Does this cause `#` in address bar and immediate jump to top on click? I just mention as previously there was no `#` and an animated slide to top.

Reply:
I noticed the `#` too, but I don't know why it appears. As for the animated slide, we can easily fix that by adding [`scroll-behavior: smooth;`](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior) to the `html` selector in CSS.

Reply:
Unless we intercept it with JS, the click will navigate to `#` and append it to URL in address bar. Not a big deal I guess.

I guess we should respect the user's animation preferences with regard to `scroll-behaviour`. Feels out of scope for this PR anyway 👍